docs/changelog.md

```diff
@@ -1900,7 +1900,7 @@ Bugfixes and other improvements for your Zulip experience.
 ### Highlights for users (since 26.1.124 / 26.2.125)
 
 * Highlight colors for code blocks now match the webapp and
-  offer more contrast, especially in night mode.
+  offer more contrast, especially in Dark theme.
```

Comment:
Bump on this part of https://github.com/zulip/zulip-mobile/pull/5595#issuecomment-1347768308:

> Let's leave `docs/changelog.md` unchanged. In the versions described by those changelog entries, the feature really was called "night mode".